feature: CLDSRV-135 integrate new bucket format

welansari opened the pull request:
- bump arsenal to integrate new bucket format for mongo metadata backend
- Added CI steps to run functional tests in both bucket version formats
- Added test for when having buckets with different vFormats at the same time

bert-e commented:
# Hello kerkesni,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/cloudserver/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue CLDSRV-135 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `8.4.3`


Please check the `Fix Version/s` of CLDSRV-135, or the target
branch of this pull request.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue CLDSRV-135 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `8.4.4`


Please check the `Fix Version/s` of CLDSRV-135, or the target
branch of this pull request.

Commit:
feature: CLDSRV-135 Bump Arsenal to 8.1.34

Commit:
feature: CLDSRV-135 Added CI steps to run tests in both formats

Added CI steps to run functional tests in both bucket metadata formats

Commit:
feature: CLDSRV-135 add tests for mixed bucket formats

Added functional tests for testing if s3 operations respect
the bucket vFormat when having buckets in different formats
